code/modules/status_system/statusEffects.dm

```diff
@@ -2161,3 +2161,23 @@
 		if(ismob(owner))
 			var/mob/M = owner
 			M.bioHolder.RemoveEffect("sims_stinky")
+
+/datum/statusEffect/flock_absorb
+	id = "flock_absorbing"
+	name = "Absorbing"
+	desc = "Please call 1800-CODER"
+	visible = FALSE
```

Comment:
Others can comment, but it might be nice to display with a little status icon of some sort

Reply:
I had a status icon on for testing and it basically just flickered `01` -> `00` most of the time, we did talk about making a loading-bar kind of thing for absorbing an item, but that's a different PR.
(basically what Flame said)